apps/explorer/src/utils/amplitude.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Types } from '@amplitude/analytics-browser';
+import { getProductAnalyticsConfig } from '@mysten/core';
+
+import { ampli } from './ampli';
+
+const IS_PROD_ENV = import.meta.env.PROD;
+
+export async function initAmplitude() {
+    const { disableCookies } = await getProductAnalyticsConfig(IS_PROD_ENV);
+    ampli.load({
```

Comment:
I looked into the whole "initialize with cookieless tracking then re-initialize with cookie tracking" and this isn't allowed by the SDK. I think this is probably the best option (I'll sync with our Amplitude ppl when I'm back from PTO)